.github/workflows/numpy2_compatibility.yml

```diff
@@ -10,10 +10,14 @@ on:
         -   '*'
 
     schedule:
-    # Run every day at 8am UTC
-    -   cron: 0 8 * * *
+    # Run Monday, Wednesday, Friday at 6am UTC
```

Comment:
Or once a week ?

Reply:
ha yeah sorry this draft was not quite meant to be reviewed but I do believe some workflows could run less often especially when they already run on the merge of a PR